example/index.html

```diff
@@ -61,6 +61,13 @@
     <p>Cat here</p>
     <img src="" data-scroll data-scroll-image="https://placeholdit.imgix.net/~text?txtsize=33&txt=Cat&w=300&h=200"/>
     <p>Vestibulum malesuada nascetur a a porta sociosqu suspendisse imperdiet enim interdum a vitae consectetur porta luctus mi magna commodo eu nisl. Natoque scelerisque lacus rutrum accumsan ac urna quis in nisl ullamcorper semper risus volutpat morbi platea a a a a aenean massa semper praesent a curae parturient suspendisse elit. Ante ac a in nisl commodo semper odio volutpat magna parturient eu mauris nisi malesuada litora parturient feugiat aptent. Id erat eu dapibus vivamus bibendum a litora consectetur vitae suspendisse a etiam varius posuere dis id nunc fusce habitant sodales tincidunt conubia in. Ornare nec massa mus a vivamus mi ridiculus a adipiscing eleifend vestibulum auctor a lacinia orci dictumst justo a adipiscing. </p>
+    <picture style="max-width: 100%; display: block">
+      <source media="(min-width: 650px)"
```

Comment:
I think we should do data-scroll-image and like how we check div or image, to determine setting background or src, we should check if tag is source and then use srcset

Reply:
Thought about that but source also works for video and srcset is also available for plain img without the picture element

Reply:
Only solution I can think about is to detect parent if it's a `source` but then `image` seems misleading.